app/views/communities/show.html.slim

```diff
@@ -1,7 +1,11 @@
 = stylesheet_link_tag 'communities'
-
-h1 class="d-flex justify-content-center"
-  = @community.title
+span
+  h1 class="d-flex justify-content-center"
+    = @community.title
+  - if user_signed_in? && current_user == @community.user
```

Comment:
Es tev atstaju par to komentaru zem cita PR, bet principā tev jāizmanto šeit view helperi. https://github.com/varvet/pundit#policies

if policy(@comunity).edit?
...

if policy(@comunity).destroy?